crates/autopilot/src/infra/database/auction/mod.rs

```diff
@@ -0,0 +1,24 @@
+use {
+    self::postgres::dto,
+    super::Database,
+    crate::domain::{self},
+};
+
+pub mod postgres;
+
+impl Database {
```

Comment:
This should probably go into `infra::persistence`? It probably doesn't matter for the domain to know if the auction is stored in a database or using other storage medium